pkg/java/jar/parse.go

```diff
@@ -159,11 +159,20 @@ func (p *Parser) parseArtifact(fileName string, size int64, r dio.ReadSeekerAt)
 		return libs, nil, nil
 	}
 
+	partialGAVInfoExist := false
+
 	manifestProps := m.properties()
+
+	if manifestProps.groupID != "" || manifestProps.artifactID != "" || manifestProps.version != "" {
```

Comment:
We can remove this check. As we discussed, we will always have the filepath for the jar. So even if GAV is missing i.e none of them are available, we can still list jar in the response with name, version attribute missing, but filepath present.

Reply:
ok, pls check

Reply:
Looks fine , just one minor comment